src/main/java/com/flansmod/common/guns/GunType.java

```diff
@@ -791,8 +791,9 @@ else if (split[0].equals("CanShootUnderwater"))
                 canShootUnderwater = Boolean.parseBoolean(split[1].toLowerCase());
             else if (split[0].equals("CanSetPosition"))
                 canSetPosition = Boolean.parseBoolean(split[1].toLowerCase());
-            else if (split[0].equals("OneHanded"))
-                oneHanded = Boolean.parseBoolean(split[1].toLowerCase());
+            else if (split[0].equals("OneHanded")) {
```

Comment:
This won't work for syncing, since types are read before PacketModConfig syncs

Reply:
Noted, will put up a change for this later

Reply:
Might be worth turning it into a get, that's definitely tidied up gun properties on other areas. That could be a goal for code cleanliness in future

Reply:
Yeah it really should overhauled to all getters/setters but that would be quite tedious haha. Switched over to a getter for oneHanded.